src/Microsoft.TemplateEngine.Abstractions/Installer/IInstaller.cs

```diff
@@ -1,44 +1,46 @@
-using Microsoft.TemplateEngine.Abstractions.TemplatesSources;
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
 using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.TemplateEngine.Abstractions.GlobalSettings;
+using Microsoft.TemplateEngine.Abstractions.TemplatesSources;
 
 namespace Microsoft.TemplateEngine.Abstractions.Installer
 {
     public interface IInstaller
     {
-        /// <summary>
-        /// Installer should determine if it can install specific <see cref="InstallRequest"/>.
-        /// Ideally it should as far as calling backend server to determine if such identifier exists.
-        /// </summary>
-        Task<bool> CanInstallAsync(InstallRequest installationRequest);
-
         /// <summary>
         /// User can specify name of specific installer to be used to install package.
         /// e.g: nuget, folder, vsix(to download from VS marketplace), npm, maven...
         /// This is useful when installer can't be determined based on <see cref="InstallRequest.Identifier"/> and <see cref="InstallRequest.Details"/>
         /// </summary>
-        string Name { get; }
+        Guid FactoryId { get; }
 
         /// <summary>
         /// User can specify name of specific installer to be used to install package.
         /// e.g: nuget, folder, vsix(to download from VS marketplace), npm, maven...
         /// This is useful when installer can't be determined based on <see cref="InstallRequest.Identifier"/> and <see cref="InstallRequest.Details"/>
         /// </summary>
-        Guid FactoryId { get; }
+        string Name { get; }
```

Comment:
Not your change: These two properties seem to have the same xml doc. Should one of these be updated?